src/Gui/DlgSettingsNavigation.cpp

```diff
@@ -78,8 +78,10 @@ void DlgSettingsNavigation::saveSettings()
     int index = ui->comboOrbitStyle->currentIndex();
     hGrp->SetInt("OrbitStyle", index);
 
+    ParameterGrp::handle hGrpNaviCube = App::GetApplication().GetParameterGroupByPath

```

Comment:
Because the widget has become a PrefComboBox you don't have to save it on your own. Instead use the widget's onSave() method.